discovery-infra/test_infra/consts.py

```diff
@@ -9,7 +9,13 @@
 NODES_REGISTERED_TIMEOUT = 600
 CLUSTER_INSTALLATION_TIMEOUT = 60 * 60   # 60 minutes
 START_CLUSTER_INSTALLATION_TIMEOUT = 360
+INSTALLING_IN_PROGRESS_TIMEOUT = 60 * 10
 VALIDATION_TIMEOUT = 6 * 60
+DISCOVERY_TIMEOUT = 60 * 15
```

Comment:
` DISCOVERY_TIMEOUT`  seems redundant with `NODES_REGISTERED_TIMEOUT` wdyt?

Reply:
Agree. removing `DISCOVERY_TIMEOUT` and raising `INSTALLING_IN_PROGRESS_TIMEOUT` to 15.